task/09-functions-n-closures-tasks.js

```diff
@@ -104,10 +112,17 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return () => {
+        for (let i = 0; i < attempts; i++)
```

Comment:
I've noticed that you don't like brackets.
There are a lot of examples when[ the approach creates](https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces) problems. So general rule is always use brackets.